packages/core/components/DateRangePicker/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { DatePicker, Icon, IconButton } from "@fluentui/react";
+import * as React from "react";
+import FileFilter from "../../entity/FileFilter";
+import { extractDatesFromRangeOperatorFilterString } from "../../entity/AnnotationFormatter/date-time-formatter";
+
+import styles from "./DateRangePicker.module.css";
+
+interface DateRangePickerProps {
+    className?: string;
+    title?: string;
+    onSearch: (filterValue: string) => void;
+    onReset: () => void;
+    currentRange: FileFilter | undefined;
+}
+
+// Because the datestring comes in as an ISO formatted date like 2021-01-02
+// creating a new date from that would result in a date displayed as the
+// day before due to the UTC offset, to account for this we can add in the offset
+// ahead of time.
+export function extractDateFromDateString(dateString?: string): Date | undefined {
+    if (!dateString) {
+        return undefined;
+    }
+    const date = new Date(dateString);
+    date.setMinutes(date.getTimezoneOffset());
+    return date;
+}
+
+const DATE_ABSOLUTE_MIN = new Date();
+DATE_ABSOLUTE_MIN.setFullYear(2000);
```

Comment:
Trivia: why 2000?  Just prettier?  I guess we can't use the founding of Cell Science, since the point is other institutions could use this to look at their own data.

Reply:
Just random :) unlikely to be a blocker for anyone